### ISSUE TITLE
[Feature] IPV6 EKS cluster blueprint for Spark

### ISSUE THREAD

**Issue by vara-bonthu** [awslabs/data-on-eks#171]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

```
- Create and IPV6 blueprint for Spark workloads with examples
- Add example to Spark Operator as a new blueprint
- This solution is to to avoid IP Exhaustion issues
```

**Comment by vara-bonthu** [comment]
@ovaleanu Please raise a PR for this issue

**Comment by askulkarni2** [comment]
Fixed in #499
